Review.java

```diff
@@ -190,4 +191,79 @@ else if(totalSentiment > 0)
       return 0;
     }
   }
+
+  public static String fakeReview(String fileName)
+  {
+    String review = textToString(fileName);
+    String fake = "";
+    
+    for(int i = 0; i < review.length()-1; i++)
+    {
+      if(review.substring(i, i+1).equals("*"))
+      {
+        i++;
+        String replace = "";
+        boolean isWord = true;
+        while(isWord)
+        {
+          replace += review.substring(i, i+1);
+          i++;
+          if(review.substring(i, i+1).equals(" "))
+          {
+            isWord = false;
+          }
+        }

```

Comment:
I think that this way of implementing the iteration was very efficient and clever.